src/main/scala/strawman/collection/immutable/TreeSet.scala

```diff
@@ -32,14 +42,11 @@ final class TreeSet[A]()(implicit val ordering: Ordering[A])
 
   // From SortedLike
   def range(from: A, until: A): TreeSet[A] = ???
-
-  // From SortedPolyTransforms
-  def map[B](f: (A) => B)(implicit ordering: Ordering[B]): TreeSet[B] = ???
```

Comment:
`ConstrainedIterablePolyTransforms` extends `IterablePolyTransforms` in order to get the linearization right for the purpose of overload resolution, so this override is no longer required.